manifests/client/mount.pp

```diff
@@ -134,7 +134,7 @@
       options  => $options_nfs,
       remounts => $remounts,
       atboot   => $atboot,
-      require  => Nfs::Functions::Mkdir[$mount]
+      require  => Nfs::Functions::Mkdir[$mount],
     }
   }
   if $owner != undef or $group != undef or $mode != undef {
```

Comment:
this if seems unecessary.
passing undef to either of those will just put them into default mode.